README.md

```diff
@@ -1 +1,55 @@
+
+### Key features
+
+- [Build, pull, and push images](https://podman-desktop.io/docs/containers/images)
+- [Build, run, and manage containers](https://podman-desktop.io/docs/containers)
+- [Compatibility with Docker and Compose](https://podman-desktop.io/docs/migrating-from-docker)
+- [Build your Kubernetes environment](https://podman-desktop.io/docs/kubernetes) 
+- [Enhance Podman Desktop by adding new features through its extensions](https://podman-desktop.io/docs/extensions)
+- [Work with LLMs using the Podman AI Lab extension](https://podman-desktop.io/docs/ai-lab)
+
+
+Learn more in [Podman Desktop Documentation](https://podman-desktop.io/docs/intro).
+
+### Quick Install
+Podman Desktop is available on Windows, macOS, and Linux. Find your favorite way to download and install it [here](https://podman-desktop.io/docs/installation). 
+
+### Getting Started
+Need to quickly learn the tool and get an overview of its capabilities, check out these links:
+
+- [Discover Podman Desktop](https://podman-desktop.io/docs/discover-podman-desktop) - Explore key features of Podman Desktop
+- [Tutorials](https://podman-desktop.io/tutorial) - Step-by-step guide to getting started with Podman Desktop
+- [Blogs](https://podman-desktop.io/blog) - Know about new releases and additional use cases of Podman Desktop
+
+
+### Your contribution matters
+
+- 🚨 [Report Issues](https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md#reporting-issues): Share bugs or improvement suggestions on GitHub, including details and reproduction steps.
+- 👾 [Fix Bugs/Develop Features](https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md#working-on-issues): Explore the issue tracker, fork the repository, make changes, and submit pull requests.
+- 🆕 [Propose Features](https://github.com/podman-desktop/podman-desktop/issues/new?template=enhancement.yml): Suggest and contribute new functionality, UI enhancements, or integrations.
+- 🛠️ [Build Extensions](https://podman-desktop.io/docs/extensions/developing): Add missing integrations by developing plug-ins using the provided guide.
+- 📝 [Improve Documentation](https://github.com/podman-desktop/podman-desktop/tree/main/website/docs): Enhance existing docs, create guides, or write tutorials.
+- 💻 [Contribute to the Website](https://github.com/podman-desktop/podman-desktop/blob/main/WEBSITE_CONTRIBUTING.md): Help improve the website’s content, design, or translations.
+- 🦭 [Become an Adopter](https://github.com/podman-desktop/podman-desktop/blob/main/ADOPTERS.md): Showcase your use of Podman Desktop by joining the adopters list.
+
+Check the [contribution guidelines](https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md) and code architecture for guidance, and join the Kubernetes Slack (#podman-desktop) to collaborate with contributors.
```

Comment:
@Firewall @slemeur , Do we have any resource that covers the code archtiecture of PD for reference here?

Reply:
We can link here I would say: https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md#code-architecture

and also https://podman-desktop.io/docs/extensions/developing for developing an extension